### PR TITLE
updated the user model,added username field and removed uniqueCode

### PR DESCRIPTION
**Pull request by johnanisere**

### CODE REVIEW

File: util/generatecode.js [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-        if (verify.length) {
-            return generateShortCode(name, Model);
-        } else {
-            return uniqueCode;
-        }
-    } catch (e) {
-        console.log(e);
-    }
-};
+// const generateShortCode = async(name, Model) => {
```
Comment (jihdeh) [inline util/generatecode.js]: you can delete this code

**Comment by jihdeh** [comment]
All looks good, can you check to make sure, places where `unqieuCode` is being used is replaced to username? Like the voting endpoint, also can you create an endpoint which we'd be hitting to check if a username is available, this would happen as the user types their username.

And there's a conflict that should be resolved before this can be merged in.
Good stuff.

**Comment by johnanisere** [comment]
oh ok..i will do that

File: server/db/seeds/user.js [hunk 2]
```diff
@@ -13,21 +12,27 @@ export default function() {
...
       phoneNumber: 2348144194590
     },
     {
       email: "xoko@x.com",
       firstName: "jake",
       lastName: "mark",
       password: "test",
-      phoneNumber: 2348069790405
+      username: "soco",
+      type: "contestant",
```
Comment (jihdeh) [inline server/db/seeds/user.js]: we can remove this type from your seeding, it's not needed

**Pull request merged**